tendrl/common/flows/base_flow.py

```diff
@@ -0,0 +1,11 @@
+import abc
```

Comment:
BaseFlow class should contain below attributes as per Tendrl schema 0.3 (https://raw.githubusercontent.com/Tendrl/node_agent/master/tendrl/node_agent/manager/tendrl_definitions_node_agent.py)

- name (str)
- atoms (ordered list)
- help (str) (requires change to current definitions, change current "description" to "help")
- enabled (bool)
- inputs (map {mandatory: (set), optional: (set)
- pre_run (ordered list)
- post_run (ordered list)
- type (str)
- uuid (str)
- parameters (dict)